content/lxd/getting-started-cli.md

```diff
@@ -97,13 +97,13 @@ For the LXD 3.0 stable release, use:
     sudo lxd.migrate
 
 #### Snap package (Arch Linux, Debian, Fedora, OpenSUSE and Ubuntu)
-LXD upstream publishes and tests a snap package which works for a number of Linux distributions.
+LXD upstream publishes and tests a [snap package](https://snapcraft.io/lxd) which works for a number of Linux distributions.
 
 The list of Linux distributions we currently test our snap for can be [found here](https://jenkins.linuxcontainers.org/job/lxd-test-snap-latest-stable/).
 
 For those distributions, you should first install snapd using [those instructions](https://snapcraft.io/docs/core/install).
 
-After that, you can install LXD with:
+After that, you can install the feature release of LXD with:
```

Comment:
"the latest stable feature release of"

Reply:
Same explanation as above, just from a different angle.
The word `stable` might sound connected to an LTS release, so to avoid confusion, I just used `feature release`.

I could add some text to the explanation [here](https://linuxcontainers.org/lxd/getting-started-cli/#choose-your-release) though, so that it is clear that all of the mentioned release branches are stable and ready for production use.